weave-js/src/core/model/graph/serialize2.ts

```diff
@@ -0,0 +1,121 @@
+// Handle serialization/deserialization of CG, for remote execution
```

Comment:
Can you add a header comment as to what is different here vs serialize?

Reply:
Added.